elm/day1/src/Iter/Extra.elm

```diff
@@ -0,0 +1,25 @@
+module Iter.Extra exposing (..)
+
+import Iter
+
+
+makeIter st f =
+    Iter.It { state = st, next = f }
+
+
+dropWhile : (a -> Bool) -> Iter.Iter a b -> Iter.Iter a b
+dropWhile pred (Iter.It it) =
+    makeIter it.state
+        (\s -> dropWhile_ pred s it.next)
+
+
+dropWhile_ pred s n =
```

Comment:
unfortunately the lib i am using for infinite lists didn't have `dropWhile` support. so i built it (maybe) 😆